contracts/intermediate-number/.gitignore

```diff
@@ -0,0 +1,15 @@
+# Build results
```

Comment:
`.gitignore` also exists in the project root. The contents are the same, so isn't this file unnecessary?
https://github.com/line/cosmwasm/blob/main/.gitignore